lms/djangoapps/support/tests/test_views.py

```diff
@@ -65,11 +66,15 @@ class SupportViewManageUserTests(SupportViewTestCase):
     Base class for support view tests.
     """
 
+    ZENDESK_URL = 'http://zendesk.example.com/'
+    UNDEFINED_ZENDESK_URL = ''
```

Comment:
The default value of ZENDESK_URL in lms/envs/common.py is already set to `''`(Ref: https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L1884). You might not need undefined Zendesk variable.